lib/pf/UnifiedApi/Controller/Config/ConnectionProfiles.pm

```diff
@@ -14,133 +14,23 @@ pf::UnifiedApi::Controller::Config::ConnectionProfiles
 
 use strict;
 use warnings;
-use Mojo::Base qw(pf::UnifiedApi::Controller::RestRoute);
+use Mojo::Base qw(pf::UnifiedApi::Controller::Config);
 use pf::ConfigStore::Profile;
+use pfappserver::Form::Config::Profile;
+use pfappserver::Form::Config::Profile::Default;
 
 has 'config_store_class' => 'pf::ConfigStore::Profile';
+has 'form_class' => 'pfappserver::Form::Config::Profile';
+has 'primary_key' => 'connection_profile_id';
```

Comment:
This module is missing the hooks that are in the Catalyst controller to create the template and perform it on all members of the cluster, etc

Likely the hooks logic in html/pfappserver/lib/pfappserver/PacketFence/Controller/Config/Profile.pm should be moved to somewhere it can be shared between both

Reply:
Also missing the deletion logic (can't delete default) that is in html/pfappserver/lib/pfappserver/Model/Config/Profile.pm